app/controllers/redo_mfa_phone_controller.rb

```diff
@@ -0,0 +1,40 @@
+class RedoMfaPhoneController < ApplicationController
+  before_action :authenticate_user!
+  before_action :ensure_redirect_url!
+
+  attr_reader :after_redo_mfa_url
+
+  def code; end
+
+  def verify
+    case MultiFactorAuth.verify_code(current_user, params[:phone_code])
+    when :ok
```

Comment:
Interesting, how come you wen with a case statement here over a variable assignment and an if?

Reply:
We don't need the result anywhere else, so I thought it was best to not name it

Reply:
fair enough